### PR TITLE
config: mark seccomp-use-default-when-empty as deprecated

### PR DESCRIPTION
**Pull request by haircommander**
as SeccompDefault FeatureGate has gone GA in upstream Kubernetes

#### What type of PR is this?

/kind deprecation
#### Does this PR introduce a user-facing change?

```release-note
Mark `seccomp-use-default-when-empty` option as deprecated, as it is replaced by the SeccompDefault FeatureGate
```

### CODE REVIEW

File: docs/crio.8.md [hunk 1]
```diff
@@ -360,15 +360,15 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--registry**="": Registry to be prepended when pulling unqualified images. Can be specified multiple times.
 
-**--root, -r**="": The CRI-O root directory. (default: /var/lib/containers/storage)
+**--root, -r**="": The CRI-O root directory. (default: /var/home/pehunt/.local/share/containers/storage)
 
-**--runroot**="": The CRI-O state directory. (default: /run/containers/storage)
+**--runroot**="": The CRI-O state directory. (default: /run/user/1000/containers)
```
Comment (sohankunkerkar) [inline docs/crio.8.md:365]: `s/pehunt/user/g`?

File: docs/crio.8.md [hunk 2]
```diff
@@ -360,15 +360,15 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--registry**="": Registry to be prepended when pulling unqualified images. Can be specified multiple times.
 
-**--root, -r**="": The CRI-O root directory. (default: /var/lib/containers/storage)
+**--root, -r**="": The CRI-O root directory. (default: /var/home/pehunt/.local/share/containers/storage)
```
Comment (sohankunkerkar) [inline docs/crio.8.md:363]: `s/pehunt/user/g?`

Comment (saschagrunert) [reply docs/crio.8.md:363]: It should be still point to `/var/lib/containers/storage`. :slightly_smiling_face:

**Review by saschagrunert — COMMENTED** [review]
Just the nit with the completion, otherwise LGTM

**Review by saschagrunert — APPROVED** [review]

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**